src/lib/viewers/controls/page/PageControls.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import IconArrowDown24 from '../icons/IconArrowDown24';
+import IconArrowUp24 from '../icons/IconArrowUp24';
+import PageControlsForm from './PageControlsForm';
+import './PageControls.scss';
+
+export type Props = {
+    onPageChange: (newPageNumber: number) => void;
+    onPageSubmit: (newPageNumber: number) => void;
+    pageCount: number;
+    pageNumber: number;
+};
+
+export default function PageControls({ onPageChange, onPageSubmit, pageCount, pageNumber }: Props): JSX.Element {
+    return (
+        <div className="bp-PageControls">
+            <button
+                className="bp-PageControls-button"
+                data-testid="bp-PageControls-previous"
+                disabled={pageNumber === 1}
+                onClick={(): void => onPageChange(pageNumber - 1)}
```

Comment:
Should this decrement logic be encapsulated here or can the calling method decide how to decrement the page number? Or is this just how it's done today and we're keeping it consistent?

Reply:
The latter!  It's how it's currently done today and we're keeping it consistent.